spec/footer_spec.rb

```diff
@@ -21,7 +25,24 @@
 
   context "Verify Footer links functionalities", :LEAGUE76_step1 do
     it 'User is redirected to the "Shipping & Payment" informations page', :step_1 do
- 
+      @footer_met.click_shipping
+      @footer_as.assert_shipping_link_works
+    end
+    it 'User is redirected to the "Returns" informations works', :step2 do
+      @footer_met.click_shipping
+      @footer_as.assert_returns_link_works
+    end
+    it 'User is redirected to the "Terms and Conditions" informations page', :step3 do
+      @footer_met.click_shipping
+      @footer_as.assert_terms_cond_link_works
+    end
+    it 'User is redirected to the "Privacy" informations page', :step4 do
+      @footer_met.click_shipping
+      @footer_as.assert_privacy_link_works
+    end
+    it 'User is redirected to the "Contact Us" informations page', :step5 do
+      @footer_met.click_shipping
+      @footer_as.assert_contact_us_link_works
     end
 
   end
```

Comment:
You need to change spec_helper.rb to adjust the  _config.app_host_ setting to the url: https://marketplace.stage.league.dev/?__t=1